letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -58,6 +58,7 @@ for arg in "$@" ; do
     --verbose)
       VERBOSE=1;;
     -[!-]*)
+      OPTIND=1
```

Comment:
Probably it's survivable to just change the template since a rebuild will happen automatically at release, but best practice is to do as Erik suggests.

We may need to expand the comment at the top of certbot-auto to explain this more extensively :)